NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 4.5.0 (IN PROGRESS)
+
+* [EDGPATRON-44](https://issues.folio.org/browse/EDGPATRON-44) Support both `circulation` `9.5` and `10.0`
+
 ## 4.4.0 2021-03-18
```

Comment:
I still think it's a waste of time to update NEWS. The release might very well be called 4.4.1 anyway.